server/src/main/java/org/apache/druid/server/initialization/jetty/JettyServerModule.java

```diff
@@ -491,6 +495,18 @@ private static int getMaxJettyAcceptorsSelectorsNum(DruidNode druidNode)
     return numServerConnector * 8;
   }
 
+  private static int getTCPAcceptQueueSize()
+  {
+    try {
+      BufferedReader in = Files.newBufferedReader(Paths.get("/proc/sys/net/core/somaxconn"));
+      return Integer.parseInt(in.readLine().trim());
+    }
+    catch (Exception e) {
+      log.warn("Unable to read /proc/sys/net/core/somaxconn, falling back to default value for TCP accept queue size");
```

Comment:
It'd be good to only log this warning on Linux. The file will never be there on macOS so the warning is always going to happen, which isn't ideal. Maybe use `System.getProperty("os.name")`?